Sources/Controllers/2.0/Settings/OAProfileAppearanceViewController.mm

```diff
@@ -72,7 +72,7 @@ - (void) commonInit
 - (void) applyLocalization
 {
     [_saveButton setTitle:OALocalizedString(@"shared_string_save") forState:UIControlStateNormal];
-    _titleLabel.text = OALocalizedString(@"new_profile");
+    _titleLabel.text = [self getProposedProfileName:_parent.name];
```

Comment:
This code has already been fixed. Please pull all the changes from ```master``` and remove this change